extra/release_tool.py

```diff
@@ -2079,6 +2086,17 @@ def figure_out_checked_out_revision(state, repo_git):
 
     return [(ref, "tag") for ref in refs]
 
+def find_repo_path(name, paths):
+    """ Try to find the git repo 'name' under some known paths.
```

Comment:
Rather than "known paths" it tries to figure out which of the "given paths" do exist.

Actually, this has nothing to do with repos. What about something like `find_existing_path(dir_name, parent_candidates)`.

Reply:
This is completely low priority.